lib/tools/adb-commands.js

```diff
@@ -965,11 +970,27 @@ methods.isDataOn = async function isDataOn () {
  *                   is not equal to '0.0'.
  */
 methods.isAnimationOn = async function isAnimationOn () {
-  let animator_duration_scale = await this.getSetting('global', 'animator_duration_scale');
-  let transition_animation_scale = await this.getSetting('global', 'transition_animation_scale');
-  let window_animation_scale = await this.getSetting('global', 'window_animation_scale');
-  return _.some([animator_duration_scale, transition_animation_scale, window_animation_scale],
-                (setting) => setting !== '0.0');
+  return (await B.all(ANIMATION_SCALE_KEYS.map(
+    async (k) => (await this.getSetting('global', k)) !== '0.0'))
+  ).includes(true);
+};
+
+/**
+ * Set animations
+ *
+ * @this {import('../adb.js').ADB}
+ * @param {number} value Animation value to set
+ * @return {Promise<boolean>} True if all set animation command succeeded.
+ * It raises an error if adb commands gets an error.
+ */
+methods.setAnimation = async function setAnimation (value) {
+  if (value < 0) {
+    value = 0;
+  }
+  for (const k of ANIMATION_SCALE_KEYS) {
+    await this.setSetting('global', k, value);
+  }
+  return true;
```

Comment:
I'm not sure we need to return any result right now. Lets just keep it unset in case we'd need to change something in the future

Reply:
makes sense